package.json

```diff
@@ -10,7 +10,13 @@
   },
   "license": "MIT",
   "engines": {},
-  "dependencies": {},
+  "dependencies": {
+    "@rollup/plugin-typescript": "^4.1.1",
```

Comment:
this should go in `devDependencies` otherwise `npm/yarn` will install this for production systems which doesn't need this

Reply:
Good catch 👍🏻